app/models/user.rb

```diff
@@ -10,6 +10,13 @@ class User < ApplicationRecord
                   against: :username,
                   using: { tsearch: { prefix: true } }
 
+  # NOTE: [@rhymes] this is adapted from config/elasticsearch/mappings/users.json
+  # Since we're moving job related fields like `available_for`, `employer_name`,
+  # and `mostly_work_with` from User to Profile, I chose not to search against them.
```

Comment:
I don't think we want to search on `User.employer_name` for example, and since we're moving "job related" fields to be optional profile fields, I removed them from the list